e2e/e2e_test.go

```diff
@@ -702,27 +717,25 @@ func Test_EVM_Native_to_EVM_Token(t *testing.T) {
 	blockTimestamp := time.Unix(int64(block.Time()), 0).UTC()
 
 	// Step 3 - Validate Lock Event was emitted with correct data
-	lockEventId := validateLockEvent(receipt, expectedLockEventLog, t)
+	lockEventId := verify.LockEvent(t, receipt, expectedLockEventLog)
 
 	// Step 4 - Verify the submitted topic messages
-	receivedSignatures := verifyTopicMessages(setupEnv, lockEventId, t)
-
-	time.Sleep(30 * time.Second)
```

Comment:
I think we will need a more stable fix, as we already sync we could have a re-try in case of timeout.